app/shop.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+from typing import List, Any
+import datetime
+from app.json_reader import data, fuel_price
+from app.car import Car
+from app.customer import Customer
+
+
+@dataclass
+class Shop:
+    name: str
+    location: List[int]
+    products: dict
+
+    def __hash__(self) -> int:
+        return hash(self.name)
+
+    @staticmethod
+    def create_shops() -> list:

```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]